<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1071</width>
      <height>791</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1061</width>
      <height>771</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Encryption">
     <attribute name="title">
      <string>Encryption</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>471</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Encryption Level</string>
      </property>
      <widget class="QPushButton" name="generateKeyBtn">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>30</y>
         <width>141</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Generate Key</string>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>188</width>
         <height>28</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QComboBox" name="dropMenu">
        <item>
         <property name="text">
          <string>RSA-1024</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RSA-768</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RSA-512</string>
         </property>
        </item>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>481</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>Public Key</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Value N</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="valueNEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>451</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Value B</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="valueBEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>451</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>471</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="savePublicKeyBtn">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>240</y>
         <width>181</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Public Key</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>0</y>
        <width>561</width>
        <height>361</height>
       </rect>
      </property>
      <property name="title">
       <string>Message</string>
      </property>
      <widget class="QTextEdit" name="messageEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>531</width>
         <height>271</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="loadMessageBtn">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>310</y>
         <width>151</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Load Message</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>350</y>
        <width>481</width>
        <height>371</height>
       </rect>
      </property>
      <property name="title">
       <string>Private Key</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Value A</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="valueAEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>451</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Value P</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="valuePEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>451</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>481</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="savePrivateKeyBtn">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>340</y>
         <width>181</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Private Key</string>
       </property>
      </widget>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>230</y>
         <width>481</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Value Q</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="valueQEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>451</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>350</y>
        <width>561</width>
        <height>371</height>
       </rect>
      </property>
      <property name="title">
       <string>Encrypted Message</string>
      </property>
      <widget class="QPushButton" name="encryptMessageBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>151</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Encrypt Message</string>
       </property>
      </widget>
      <widget class="QPushButton" name="saveEncryptedBtn">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>20</y>
         <width>221</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Encrypted Message</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="encryptedMessageEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>531</width>
         <height>301</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Decryption">
     <attribute name="title">
      <string>Decryption</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1051</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Keys</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Value A</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loadPrivateKeyBtn">
       <property name="geometry">
        <rect>
         <x>850</x>
         <y>20</y>
         <width>181</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Load Private Key</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="valueAEdit2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>1011</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>140</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Value N</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loadPublicKeyBtn">
       <property name="geometry">
        <rect>
         <x>850</x>
         <y>130</y>
         <width>181</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Load Public Key</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="valueNEdit2">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>160</y>
         <width>491</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="Line" name="line_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>1051</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QTextEdit" name="valueBEdit2">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>160</y>
         <width>511</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>140</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Value B</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>511</width>
        <height>491</height>
       </rect>
      </property>
      <property name="title">
       <string>Encrypted Text</string>
      </property>
      <widget class="QPushButton" name="loadEncryptedMessageBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>231</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Load Encrypted Message</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="encryptedMessageEdit2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>491</width>
         <height>411</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_14">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>230</y>
        <width>531</width>
        <height>491</height>
       </rect>
      </property>
      <property name="title">
       <string>Decrypted Text</string>
      </property>
      <widget class="QPushButton" name="DecryptMessageBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>161</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Decrypt Message</string>
       </property>
      </widget>
      <widget class="QPushButton" name="saveDecryptedMessageBtn">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>30</y>
         <width>221</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Decrypted Message</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="messageEdit2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>501</width>
         <height>411</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
